### PR TITLE
adding more descriptive chair role and reformat

### PR DESCRIPTION
**Pull request by parispittman**
Answers #4577 - issue has the where the new information came from (other gov doc sources), reformatting and taking out duplicate information preventing us from having to update in multiple places, and adding very minor content. It will be best to read the original doc first entirely and then the new doc so you can see how things have moved around.

Open-ended questions and discussions pulled from the comments:
- the term member is bloated and needs to be changed. most folks associate member with sig member/contributor and not a role regardless of whether the term member is defined here. 
https://github.com/kubernetes/steering/issues/160

### CODE REVIEW

File: committee-steering/governance/sig-governance.md [hunk 1]
```diff
@@ -1,62 +1,135 @@
...
+  approve presented content but *MAY* delegate to others to create material and
+  present  
+  - *MUST* Ensure meetings are recorded and made available
+  - *MUST* Report activity with the community via k-dev mailing list at least
+  once a quarter (slides, video from kubecon, etc)
+  - *MUST* Coordinate sponsored working group updates to the SIG and the wider
+  community  
+ - *MUST* coordinate communication and be a connector with other community
+ groups like SIGs and the Steering Committee but may delegate the actual
+ communication and creation of to other community members where appropriate
```
Comment (spiffxp) [inline committee-steering/governance/sig-governance.md:127]: creation of what?

File: committee-steering/governance/sig-governance.md [hunk 2]
```diff
@@ -1,62 +1,135 @@
...
+  - *MUST* Report activity with the community via k-dev mailing list at least
+  once a quarter (slides, video from kubecon, etc)
+  - *MUST* Coordinate sponsored working group updates to the SIG and the wider
+  community  
+ - *MUST* coordinate communication and be a connector with other community
+ groups like SIGs and the Steering Committee but may delegate the actual
+ communication and creation of to other community members where appropriate  
+- *MUST* provide quarterly updates through our community channels: twice a year
+to kubernetes-dev@googlegroups.com mailing list and twice a year presenting at
+the monthly community meeting
```
Comment (spiffxp) [inline committee-steering/governance/sig-governance.md:130]: This is in conflict with the other bullets that refer to quarterly e-mail updates, I feel like this should take precedence and be moved (or re-stated) to the SIG responsibilities

File: committee-steering/governance/sig-governance.md [hunk 3]
```diff
@@ -1,62 +1,135 @@
...
+  - *MUST* Coordinate sponsored working group updates to the SIG and the wider
+  community  
+ - *MUST* coordinate communication and be a connector with other community
+ groups like SIGs and the Steering Committee but may delegate the actual
+ communication and creation of to other community members where appropriate  
+- *MUST* provide quarterly updates through our community channels: twice a year
+to kubernetes-dev@googlegroups.com mailing list and twice a year presenting at
+the monthly community meeting  
+
+[### Tech Lead]
```
Comment (spiffxp) [inline committee-steering/governance/sig-governance.md:132]:
```suggestion
### Tech Lead
```

File: committee-steering/governance/sig-governance.md [hunk 4]
```diff
@@ -1,62 +1,135 @@
...
+- Keep up-to-date meeting notes, linked from the SIG's page in the community
+repo
+- Ensure other SIGs and community members can find notes of meetings,
+discussions, designs, and decisions
+- Record meetings and make them publicly available on the
+[Kubernetes Community YouTube playlist]
+- Report activity with the community via the kubernetes-dev mailing list at
+least once a quarter. Whichever format the SIG uses should _always_ be reported
+to the kubernetes-dev mailing list as this is the list the community depends on
+for SIG updates.
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:22]: this was the new process that was PR'ed in separately from Jorge when we went to monthly vs weekly cc/ @castrojo

File: committee-steering/governance/sig-governance.md [hunk 5]
```diff
@@ -1,62 +1,135 @@
...
-- Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc.
-- Use the [forums provided] as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
-
-The process for setting up a SIG or Working Group (WG) is listed in the [sig-wg-lifecycle] document.
+- Meet regularly, at least for 30 minutes every 3 weeks, except November and
+December
+- Keep up-to-date meeting notes, linked from the SIG's page in the community
+repo
+- Ensure other SIGs and community members can find notes of meetings,
+discussions, designs, and decisions
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:16]: I'm open to completely removing. Can't really think of how to elaborate but will leave this open for others that might.

File: committee-steering/governance/sig-governance.md [hunk 6]
```diff
@@ -1,62 +1,135 @@
...
+- Report activity with the community via the kubernetes-dev mailing list at
+least once a quarter. Whichever format the SIG uses should _always_ be reported
+to the kubernetes-dev mailing list as this is the list the community depends on
+for SIG updates.  
+  - Each SIG is assigned an update during the monthly community meeting
+  throughout the year. The meeting host will publish the notes to the
+  kubernetes-dev mailing list with the update.  
+  - If a SIG is doing an update during KubeCon + CloudNativeCon, the SIG Chairs
+  should ensure that a link to the recording of the video is sent to the
+  kubernetes-dev mailing list.
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:28]: I think this was also apart of @castrojo changes with going from weekly to monthly

File: committee-steering/governance/sig-governance.md [hunk 7]
```diff
@@ -1,62 +1,135 @@
...
+  mailing list.
+- Participate in release planning meetings and retrospectives, and burndown
+meetings, as needed
+- Ensure related work happens in a project-owned github org and repository, with
+ code and tests explicitly owned and supported by the SIG, including issue
+ triage, PR reviews, test-failure response, bug fixes, etc.
+- Use the [forums provided] as the primary means of working, communicating, and
+collaborating, as opposed to private emails and meetings  
+- SIG meets at least once every three weeks on zoom with agenda in meeting notes
+ except November and December
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:39]: will remove

File: committee-steering/governance/sig-governance.md [hunk 8]
```diff
@@ -1,62 +1,135 @@
...
+- SIG overview and deep-dive sessions organized for KubeCon/CloudNativeCon
+- SIG updates to Kubernetes community meeting on a regular basis
+- Contributing instructions defined in the SIGs CONTRIBUTING.md
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:42]: either or? i'll be more clear here. the emphasis is on something rather than nothing.

File: committee-steering/governance/sig-governance.md [hunk 9]
```diff
@@ -1,62 +1,135 @@
...
+[sig-wg-lifecycle] document.
 
 ## Roles
 
 ### Notes on Roles
 
-Within this section "member" refers to a member of a Chair, Tech Lead or
-Subproject Owner Role.  (this different from a SIG or Organization Member).  
+*Within this section "member" refers to a member of a Chair, Tech Lead or
+Subproject Owner Role.  (this different from a SIG or [Organization Member]).*
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:54]: I'm ok with that too

[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:54]: i thought the previous committee wanted to stray from using the term lead though

File: committee-steering/governance/sig-governance.md [hunk 10]
```diff
@@ -1,62 +1,135 @@
...
+- Number: 2-3
+- Membership tracked in [sigs.yaml]  
+
+Run operations and processes governing the SIG:
+  - *SHOULD* define how priorities and commitments are managed and delegate to
+  other members as needed  
+  - *SHOULD* drive charter changes, including creation, to get community buy-in
+  but *MAY* delegate content creation to community members
+  - *MUST* identify, track, and maintain the groups features for current release
+  and serve as point of contact for the release team
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:108]: ack will update

File: committee-steering/governance/sig-governance.md [hunk 11]
```diff
@@ -1,62 +1,135 @@
...
+  - *MUST* identify, track, and maintain the groups features for current release
+  and serve as point of contact for the release team  
+  - *MAY* delegate the creation of a SIG roadmap to other members
+  - *MUST* organize a main group meeting and make sure sigs.yaml is up to date
+  including subprojects and their meeting information but *SHOULD* delegate the
+  need for subproject meetings to owners  
+  - *SHOULD* facilitate meetings but *MAY* delegate to other members or future
+  chairs/chairs in training
+  - *MUST* Create and maintain a CONTRIBUTING.md document in the appropriate SIG
+  folder. *MAY* ask for help from community members to update / send in PRs.
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:116]: the last part is new : may ask for help; the first part is legacy

File: committee-steering/governance/sig-governance.md (hunk 1 above)
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:127]: oops forgot the word content

...and creation of content to other...

File: committee-steering/governance/sig-governance.md (hunk 2 above)
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:130]: SIGs were previously doing one update a quarter via the weekly community meeting. (4 updates a year)

File: committee-steering/governance/sig-governance.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:16]: ok found the reference - this is located in /community/governance.md so I carried that over from there. do you think we should remove from there as well in addition to this? i think they should both be added or removed but not one or the other.

File: committee-steering/governance/sig-governance.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (castrojo) [reply committee-steering/governance/sig-governance.md:22]: We've been having the host of the meeting posting the full notes directly to -dev after each meeting now which does include the SIG updates.  In hindsight the wording here makes it sound like you'd be getting the updates from the SIGs when we really mean that we're doing a much better job of getting the full meeting notes out to -dev regularly. 

Since moving to the monthly format we have not yet had a SIG need to use the "post your youtube video to the list" or do any of the other extra methods needed to post to -dev. We mostly added these as options in the case SIGs would need to be more flexible from missing a community meeting.

File: committee-steering/governance/sig-governance.md [hunk 12]
```diff
@@ -1,62 +1,135 @@
...
+meetings, as needed
+- Ensure related work happens in a project-owned github org and repository, with
+ code and tests explicitly owned and supported by the SIG, including issue
+ triage, PR reviews, test-failure response, bug fixes, etc.
+- Use the [forums provided] as the primary means of working, communicating, and
+collaborating, as opposed to private emails and meetings  
+- SIG meets at least once every three weeks on zoom with agenda in meeting notes
+ except November and December
+- SIG overview and deep-dive sessions organized for KubeCon/CloudNativeCon
+- SIG updates to Kubernetes community meeting on a regular basis
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:41]: removed

File: committee-steering/governance/sig-governance.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (castrojo) [reply committee-steering/governance/sig-governance.md:28]: These options were added to give SIG Chairs the flexibility to use their KubeCon presentation as status update to the project if they had missed one during the normal community meeting process, so it's probably a good thing this hasn't been needed yet but the wording could perhaps be clearer.

File: committee-steering/governance/sig-governance.md [hunk 13]
```diff
@@ -1,62 +1,135 @@
...
+to the kubernetes-dev mailing list as this is the list the community depends on
+for SIG updates.  
+  - Each SIG is assigned an update during the monthly community meeting
+  throughout the year. The meeting host will publish the notes to the
+  kubernetes-dev mailing list with the update.  
+  - If a SIG is doing an update during KubeCon + CloudNativeCon, the SIG Chairs
+  should ensure that a link to the recording of the video is sent to the
+  kubernetes-dev mailing list.  
+  - YouTube presentation or mailing list update sent to the kubernetes-dev
+  mailing list.
```
[the remark this answers is not in the document]
Comment (castrojo) [reply committee-steering/governance/sig-governance.md:30]: "A YouTube recording of a SIG status update or an email status update to the kubernetes-dev mailing list." perhaps?

File: committee-steering/governance/sig-governance.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:116]: changed it to "must ensure there is a maintained.."

**Comment by parispittman** [comment]
@spiffxp made all of the minor changes you suggested and left some of the others with open ended questions/discussion.

File: committee-steering/governance/sig-governance.md [hunk 14]
```diff
@@ -178,16 +235,16 @@ Issues impacting multiple subprojects in the SIG should be resolved by either:
...
-[SIG Charter process]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
-[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+[SIG Charter process]: https://git.k8s.io/community/committee-steering/governance/README.md
+[Kubernetes Charter README]: https://git.k8s.io/community/committee-steering/governance/README.md
 [Embargo Policy]: https://git.k8s.io/security/private-distributors-list.md#embargo-policy
 [SECURITY_CONTACTS]: https://github.com/kubernetes/kubernetes-template-project/blob/master/SECURITY_CONTACTS
 [sig-wg-lifecycle]: /sig-wg-lifecycle.md
 [community members]: /community-membership.md
 [Kubernetes Community YouTube playlist]: https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg
+[Organizational Members]: https://git.k8s.io/community/community-membership.md#established-community-members
```
Comment (nikhita) [inline committee-steering/governance/sig-governance.md:250]:
```suggestion
[Organization Members]: https://git.k8s.io/community/community-membership.md#established-community-members
```

File: committee-steering/governance/sig-governance.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (nikhita) [reply committee-steering/governance/sig-governance.md:16]: I'd be ok with removing at both places. We already mention that SIGs need to have meeting notes and that they need to be linked in sigs.yaml.

File: committee-steering/governance/sig-governance.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (nikhita) [reply committee-steering/governance/sig-governance.md:22]: > Since moving to the monthly format we have not yet had a SIG need to use the "post your youtube video to the list" or do any of the other extra methods needed to post to -dev. We mostly added these as options in the case SIGs would need to be more flexible from missing a community meeting.

Thanks, this helps clarify! Maybe we could explicitly state here that point 2 and 3 are optional?

File: committee-steering/governance/sig-governance.md [hunk 15]
```diff
@@ -1,60 +1,134 @@
...
+  - YouTube presentation or mailing list update sent to the kubernetes-dev
+  mailing list.
+- Participate in release planning meetings and retrospectives, and burndown
+meetings, as needed
+- Ensure related work happens in a project-owned github org and repository, with
+ code and tests explicitly owned and supported by the SIG, including issue
+ triage, PR reviews, test-failure response, bug fixes, etc.
+- Use the [forums provided] as the primary means of working, communicating, and
+collaborating, as opposed to private emails and meetings  
+- SIG overview and deep-dive sessions organized for KubeCon/CloudNativeCon
```
Comment (nikhita) [inline committee-steering/governance/sig-governance.md:38]: Can we rephrase this as "Organize SIG overview and deep-dive sessions at KubeCon/CloudNativeCon"?

I'm wondering if we need some sort a phrase here that this optional? I'm not sure if _all_ SIGs have necessarily had sessions in the past :thinking: It's also not necessary to have a deep-dive session...

File: committee-steering/governance/sig-governance.md [hunk 16]
```diff
@@ -1,60 +1,134 @@
...
+- Contributing instructions (CONTRIBUTING.md) defined in the SIGs folder located
+ in the Kubernetes/community repo
```
Comment (nikhita) [inline committee-steering/governance/sig-governance.md:40]: Can we rephrase this to start with something like "Ensure contributing instructions  (CONTRIBUTING.md) defined in..."?

Most SIGs don't have this file right now. Is this something that SIGs _MUST_ do or is it something SIGs _MAY_?

File: committee-steering/governance/sig-governance.md [hunk 17]
```diff
@@ -1,60 +1,134 @@
...
+section in addition
+- Number: 2-3
+- Membership tracked in [sigs.yaml]  
+
+Run operations and processes governing the SIG:
+  - *SHOULD* define how priorities and commitments are managed and delegate to
+  other members as needed  
+  - *SHOULD* drive charter changes (including creation) to get community buy-in
+  but *MAY* delegate content creation to community members
+  - *SHOULD* identify, track, and maintain the groups features for current release
```
Comment (nikhita) [inline committee-steering/governance/sig-governance.md:105]: I'm probably missing something but what does "groups feature" mean here?

File: committee-steering/governance/sig-governance.md [hunk 18]
```diff
@@ -1,60 +1,134 @@
...
+    and serve as point of contact for the release team, but *MAY* delegate to
+    another member to fulfill these responsibilities
+  - *MAY* delegate the creation of a SIG roadmap to other members
+  - *MUST* organize a main group meeting and make sure [sigs.yaml] is up to date
+  including subprojects and their meeting information but *SHOULD* delegate the
+  need for subproject meetings to subproject owners  
+  - *SHOULD* facilitate meetings but *MAY* delegate to other members or future
+  chairs/chairs in training
```
Comment (nikhita) [inline committee-steering/governance/sig-governance.md:113]: > chairs in training

:100:

File: committee-steering/governance/sig-governance.md [hunk 19]
```diff
@@ -1,60 +1,134 @@
...
+  - *MUST* Organize KubeCon Intros and Deep Dives with CNCF Event staff and
+  approve presented content but *MAY* delegate to others to create material and
+  present  
+  - *MUST* Ensure meetings are recorded and made available
+  - *MUST* Report activity with the community via k-dev mailing list at least
+  once a quarter (slides, video from kubecon, etc)
+  - *MUST* Coordinate sponsored working group updates to the SIG and the wider
+  community  
+ - *MUST* coordinate communication and be a connector with other community
+ groups like SIGs and the Steering Committee but may delegate the actual
```
Comment (nikhita) [inline committee-steering/governance/sig-governance.md:126]:
```suggestion
 groups like SIGs and the Steering Committee but *MAY* delegate the actual
```

File: committee-steering/governance/sig-governance.md [hunk 20]
```diff
@@ -1,60 +1,134 @@
...
+amongst members. This *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
 
 ### Chair
 
 - Chair
-  - Run operations and processes governing the SIG
-  - Number: 2-3
-  - Membership tracked in [sigs.yaml]
+If no tech lead role is present, Chair assumes responsibilities from #tech-lead
```
Comment (nikhita) [inline committee-steering/governance/sig-governance.md:95]:
```suggestion
  - If no tech lead role is present, Chair assumes responsibilities from #tech-lead
```

File: committee-steering/governance/sig-governance.md (hunk 17 above)
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:105]: changing to SIGs enhancements - wdyt?

File: committee-steering/governance/sig-governance.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:20]: PTAL 

- Report activity with the community via the kubernetes-dev mailing list at
least once a quarter. Whichever format the SIG uses should _always_ be reported
to the kubernetes-dev mailing list as this is the list the community depends on
for SIG updates.  
  - Each SIG is assigned an update during the monthly community meeting
  throughout the year. The meeting host will publish the notes to the
  kubernetes-dev mailing list with the update.  
  - Due to limited opportunities (twice a year) at the community meeting to
  present, your SIG will need to figure out two other options to deliver on SIG 
  updates:
    1. During KubeCon + CloudNativeCon or  
    2. YouTube presentation or mailing list update sent to the kubernetes-dev
  mailing list.

File: committee-steering/governance/sig-governance.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md]: reworked this section PTAL

File: committee-steering/governance/sig-governance.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md]: reworked this section based on nikhitas feedback as well PTAL

File: committee-steering/governance/sig-governance.md (hunk 16 above)
Comment (parispittman) [reply committee-steering/governance/sig-governance.md]: PTAL

**Comment by parispittman** [comment]
@spiffxp @nikhita PTAL. made suggested changes and left some discussions.

File: committee-steering/governance/sig-governance.md [hunk 21]
```diff
@@ -1,62 +1,135 @@
...
-  majority of SIG Members or Subproject Contributors (as applicable).
-- Members *MAY* select additional members through a [super-majority] vote amongst members. This
-  *SHOULD* be supported by a majority of SIG Members or Subproject Contributors (as applicable).
+- Members of a role *SHOULD* remove any other members that have not communicated
+ a leave of absence and either cannot be reached for more than 1 month or are
+ not fulfilling their documented responsibilities for more than 1 month.
+  This may be done through a [super-majority] vote of members, or if there are
+  not enough *active* members to get a super-majority of votes cast, then
+  removal may occur through a [super-majority] vote between Chairs, Tech Leads
+  and Subproject Owners.
```
[the remark this answers is not in the document]
Comment (spiffxp) [reply committee-steering/governance/sig-governance.md:72]: The second vote is redundant if "member" here still means "a Chair, Tech Lead, or Subproject Owner Role".  Is the first vote supposed to be from SIG members? Do we define SIG member as a member of their ML?

**Comment by spiffxp** [comment]
I'm fine with everything here now except the language around voting to remove someone.

File: committee-steering/governance/sig-governance.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:54]: scopecreep-y idea: what about the term "core SIG roles" or "key SIG roles" as an umbrella to describe sig members and/or organization members?

[the remark this answers is not in the document]
Comment (mrbobbytables) [reply committee-steering/governance/sig-governance.md:54]: +1 on lead or leader. It makes it clear who it's targeting without necessarily having to read the 1st line that defines a member.

File: committee-steering/governance/sig-governance.md [hunk 22]
```diff
@@ -1,60 +1,136 @@
...
+lazy consensus amongst other members with fallback on majority vote to accept
+proposal.  The candidate *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
+- Members *MAY* select additional members through a [super-majority] vote
+amongst members. This *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
 
 ### Chair
 
 - Chair
```
Comment (mrbobbytables) [inline committee-steering/governance/sig-governance.md:96]:
```suggestion

```
With the heading and new break out this additional `Chair` probably isn't needed.

File: committee-steering/governance/sig-governance.md [hunk 23]
```diff
@@ -1,60 +1,136 @@
...
-  - Run operations and processes governing the SIG
-  - Number: 2-3
-  - Membership tracked in [sigs.yaml]
+
+- Number: 2-3
+- Membership tracked in [sigs.yaml]  
+  - If no tech lead role is present, Chair assumes responsibilities from #tech-lead
+section in addition  
+Run operations and processes governing the SIG:
```
Comment (mrbobbytables) [inline committee-steering/governance/sig-governance.md:102]:
```suggestion
- Run operations and processes governing the SIG:
```
I think the beginning was missing a `-` from looking at the spacing.

File: committee-steering/governance/sig-governance.md [hunk 24]
```diff
@@ -1,60 +1,136 @@
...
+   to another member to fulfill these responsibilities
+  - *MAY* delegate the creation of a SIG roadmap to other members
+  - *MUST* organize a main group meeting and make sure [sigs.yaml] is up to date
+  including subprojects and their meeting information but *SHOULD* delegate the
+  need for subproject meetings to subproject owners  
+  - *SHOULD* facilitate meetings but *MAY* delegate to other members or future
+  chairs/chairs in training
+  - *MUST* ensure there is a maintained CONTRIBUTING.md document in the
+  appropriate SIG folder. *MAY* ask for help from community members to update /
+  send in PRs.
```
Comment (mrbobbytables) [inline committee-steering/governance/sig-governance.md:118]: This differs from the above slightly in that the SIG only needs their own `CONTRIBUTING.md` if it differs from the practices outlined in the contributor guide.

Maybe
```
*MUST* ensure there is a maintained CONTRIBUTING.md document in the appropriate SIG folder *IF*
practices differ the guidelines outlined in the [contributor guide] and [devel] folder. *MAY* ask for help
from community members to update /send in PRs.
```

For reference here is the snippet from the above:
```
- Ensure contributing instructions (CONTRIBUTING.md) are defined in the SIGs
folder located in the Kubernetes/community repo if the groups contributor steps
and experience are different or more in-depth than the documentation listed in
the general [contributor guide] and [devel] folder.  
```

File: committee-steering/governance/sig-governance.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (lavalamp) [reply committee-steering/governance/sig-governance.md:54]: How about "officer"? "member" sounds like someone who just attends the SIG.

File: committee-steering/governance/sig-governance.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (lavalamp) [reply committee-steering/governance/sig-governance.md:72]: Yeah, I think "members" as a term makes this confusing.

File: committee-steering/governance/sig-governance.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (ehashman) [reply committee-steering/governance/sig-governance.md:54]: I've mostly heard the term "Lead" used.

[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:54]: I added this to the top in the issue under open-ended comments and discussion so I don't miss in the rest of the comments. I need more brain power than typo fixes right now on this one hehe.

File: committee-steering/governance/sig-governance.md [hunk 25]
```diff
@@ -1,60 +1,136 @@
...
+  mailing list.
+- Participate in release planning meetings and retrospectives, and burndown
+meetings, as needed
+- Ensure related work happens in a project-owned github org and repository, with
+ code and tests explicitly owned and supported by the SIG, including issue
+ triage, PR reviews, test-failure response, bug fixes, etc.
+- Use the [forums provided] as the primary means of working, communicating, and
+collaborating, as opposed to private emails and meetings  
+- Present the opportunity to the group to have a SIG overview/intro and/or
+deep-dive sessions organized for KubeCon/CloudNativeCon
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:38]: what if i list:
"Present a SIG Intro or Deep Dive at KubeCon..."

and then move this down to chair duties?

File: committee-steering/governance/sig-governance.md [hunk 26]
```diff
@@ -1,60 +1,136 @@
...
+
+#### Requirements
+- Members *MUST* be [community members] to be eligible to hold a leadership role
+  within a SIG.
+- SIGs *MAY* prefer various levels of domain knowledge depending on the
+role. This should be documented.  
+- People management interests - there's a lot of us!    
+
+#### Escalations
+- Membership disagreements *MAY* be escalated to the SIG Chairs. SIG Chair
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:82]: i think this means if there is a disagreement amongst tech leads and/or subproject owners as tech leads/spo in this docs case = members

File: committee-steering/governance/sig-governance.md [hunk 27]
```diff
@@ -1,60 +1,136 @@
...
+  other members as needed  
+  - *SHOULD* drive charter changes (including creation) to get community buy-in
+  but *MAY* delegate content creation to community members
+  - *SHOULD* identify, track, and maintain the SIGs enhancements for current
+  release and serve as point of contact for the release team, but *MAY* delegate
+   to another member to fulfill these responsibilities
+  - *MAY* delegate the creation of a SIG roadmap to other members
+  - *MUST* organize a main group meeting and make sure [sigs.yaml] is up to date
+  including subprojects and their meeting information but *SHOULD* delegate the
+  need for subproject meetings to subproject owners
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md:113]: yes - we used to include it at the top in the general sig operations area and the cadence in the chair duties. One seemed to be out of sync with the other so I included in just the operational sig section as a canonical source. what would make better sense? something like:
"*MUST* organize a main group meeting as according to the guidelines listed at the start of this doc..." wdyt

**Comment by parispittman** [comment]
addressed most comments with the exception of the member term which will need to be scrubbed from the doc and added that to the issue body. seems to be the only major edit needed at the moment.

File: committee-steering/governance/sig-governance.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (logicalhan) [reply committee-steering/governance/sig-governance.md:54]: I like officer, the term 'member' is a bit confusing.

File: committee-steering/governance/sig-governance.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (logicalhan) [reply committee-steering/governance/sig-governance.md:72]: +1

**Comment by nikhita** [comment]
After the member term is fixed (it really is very confusing!) and some minor nits (like https://github.com/kubernetes/community/pull/4716#discussion_r414113205) are addressed, looks good to me overall. :+1: 

xref https://github.com/kubernetes/steering/issues/160

**Comment by parispittman** [comment]
47 instances of the term "member" in the doc before editing; 9 after. replaced with lead(s) and/or "sig contributors" depending on the context. 

PTAL everyone that commented on the term member (ex: @lavalamp @nikhita @spiffxp @mrbobbytables @logicalhan @dims @ehashman sorry if I missed someone)

i think i have one more discussion/comment with @ehashman but everything else is clear.

File: committee-steering/governance/sig-governance.md [hunk 28]
```diff
@@ -175,19 +236,20 @@ Issues impacting multiple subprojects in the SIG should be resolved by either:
...
+[Kubernetes Charter README]: https://git.k8s.io/community/committee-steering/governance/README.md
 [Embargo Policy]: https://git.k8s.io/security/private-distributors-list.md#embargo-policy
 [SECURITY_CONTACTS]: https://github.com/kubernetes/kubernetes-template-project/blob/master/SECURITY_CONTACTS
 [sig-wg-lifecycle]: /sig-wg-lifecycle.md
-[community members]: /community-membership.md
+["member" on our contributor ladder]: /community-membership.md
 [Kubernetes Community YouTube playlist]: https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg
+[contributor guide]: /contributors/guide/README.md
+[devel]: /contributors/devel/README.md
+[#tech-lead]: (#Tech-Lead)
```
Comment (mrbobbytables) [inline committee-steering/governance/sig-governance.md:255]:
```suggestion
[#tech-lead]: #Tech-Lead
```

Fixes the link 👍

**Comment by dims** [comment]
LGTM !!

/approve

**Comment by nikhita** [comment]
LGTM :tada: 

/hold
explicitly adding hold so it isn't accidentally merged before lgtm from @ehashman @mrbobbytables and other steering members

**Comment by enj** [comment]
I gave this a quick scan.  LGTM, though I am not strongly opinionated in this area 😀

File: committee-steering/governance/sig-governance.md [hunk 29]
```diff
@@ -1,60 +1,136 @@
...
+The process for setting up a SIG or Working Group (WG) is listed in the
+[sig-wg-lifecycle] document.
 
 ## Roles
 
 ### Notes on Roles
 
-Within this section "member" refers to a member of a Chair, Tech Lead or
-Subproject Owner Role.  (this different from a SIG or Organization Member).  
+*Within this section "Lead" refers to someone who is a member of the union
```
Comment (lachie83) [inline committee-steering/governance/sig-governance.md:51]: I think this is stray star

```suggestion
Within this section "Lead" refers to someone who is a member of the union
```

File: committee-steering/governance/sig-governance.md [hunk 30]
```diff
@@ -1,60 +1,136 @@
 # SIG Roles and Organizational Governance
 
-This charter adheres to the conventions described in the [Kubernetes Charter README]. It will be updated as needed to meet the current needs of the Kubernetes project.
+This charter adheres to the conventions described in the [Kubernetes Charter README].
+It will be updated as needed to meet the current needs of the Kubernetes project.
 
-In order to standardize Special Interest Group efforts, create maximum transparency, and route contributors to the appropriate SIG, SIGs should follow these guidelines:
+In order to standardize Special Interest Group efforts, create maximum
+transparency, and route contributors to the appropriate SIG, SIGs should follow
```
Comment (lachie83) [inline committee-steering/governance/sig-governance.md:7]: Should we simply link any of the required items that are found here and under the `### Chair` section rather duplicate in both places?

Comment (parispittman) [reply committee-steering/governance/sig-governance.md:7]: im 100% on board with that. 

message for the audience: spoke to lachie in DMs on slack. we are going to merge and iterate on this section. before completely removing / reorganizing this part, i need to make sure the details are covered below, too. .

File: committee-steering/governance/sig-governance.md [hunk 31]
```diff
@@ -1,60 +1,136 @@
...
-  *SHOULD* be supported by a majority of SIG Members or Subproject Contributors (as applicable).
+- Leads going on leave for 1-3 months *MAY* work with other
+  Leads to identify a temporary replacement.
+- Leads of a role *SHOULD* remove any other leads or roles that have not
+communicated a leave of absence and either cannot be reached for more than 1
+month or are not fulfilling their documented responsibilities for more than 1 month.
+  This may be done through a [super-majority] vote of Leads, or if there are
+  not enough *active* Leads to get a super-majority of votes cast, then
+  removal may occur through a [super-majority] vote between Chairs, Tech Leads
+  and Subproject Owners.
```
Comment (cblecker) [inline committee-steering/governance/sig-governance.md:72]: This is confusing and may need an editorial followup.. if Leads are the union of Chairs+TL+SOs, then isn't this saying the same group twice?

File: committee-steering/governance/sig-governance.md [hunk 32]
```diff
@@ -1,60 +1,136 @@
...
+  appropriate SIG folder if the contributor experience or on-boarding knowledge
+  is different than in the general [contributor guide]. *MAY* delegate to
+  contributors to create or update.
+  - *MUST* organize KubeCon/CloudNativeCon Intros and Deep Dives with CNCF Event
+   staff and approve presented content but *MAY* delegate to other contributors
+   to create material and present  
+  - *MUST* ensure meetings are recorded and made available
+  - *MUST* report activity with the community via k-dev mailing list at least
+  once a quarter (slides, video from kubecon, etc)
+  - *MUST* coordinate sponsored working group updates to the SIG and the wider
```
Comment (cblecker) [inline committee-steering/governance/sig-governance.md:125]: There's some weird whitespacing  we can follow up on as well
